custom/icds/static/icds/js/location_rationalization/download.js

```diff
@@ -0,0 +1,13 @@
+hqDefine('icds/js/location_rationalization/download', [
+    'jquery',
+    'locations/js/utils',
+    'hqwebapp/js/widgets', // using select2/dist/js/select2.full.min for ko-select2 on location select
+], function (
+    $,
+    locationUtils
+) {
+    'use strict';
+    $(function () {
+        locationUtils.enableLocationSearchSelect();
```

Comment:
that sounds good to me.
All pages that i have added this, i think this would work as expected.
Just thinking if this would delay the setup of the locations dropdown in case the DOM takes time to load or any other scenario i can't think of?
making the change nonetheless.

Reply:
since its already under location i think it's fine to just name it "search"

Init changes https://github.com/dimagi/commcare-hq/pull/26865/commits/4076e3d89cd312111468aec948fbda4fd155bbe8

Rename https://github.com/dimagi/commcare-hq/pull/26865/commits/470add008090d06e0f09627632017959b08d9e6a

Reply:
cool, thank you.

> Just thinking if this would delay the setup of the locations dropdown in case the DOM takes time to load or any other scenario i can't think of?

I think it's possible that sometimes the search will get initialized a moment later...but I think the old code is subject to a race condition, where it's possible for `enableLocationSearchSelect` (which is typically run in a document ready handler) to be run before `utils.js` has run its own document ready handler and assigned values to `show_inactive`, `location_search_url`, and `locs`. So I think the new code is safer.

Reply:
Can you confirm that the changes i made for this were all the places this needs to be changed?

Reply:
Changes look good to me!